### PR TITLE
WorkflowRun stats in workflow details page for analytics

### PR DESCRIPTION
**Pull request by arkajyotiMukherjee**
Signed-off-by: arkajyotiMukherjee <arko@chaosnative.com>

## Proposed changes

- Includes workflow run stats for all or specified workflows

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Comment by arkajyotiMukherjee** [comment]
This can go in Beta9 @rajdas98

File: litmus-portal/graphql-server/pkg/analytics/handler/handler.go [hunk 1]
```diff
@@ -966,3 +966,204 @@ func GetScheduledWorkflowStats(projectID string, filter model.TimeFrequency, sho
...
+	// Count Succeeded workflowRuns
+	succeededWorkflowRuns := bson.A{
+		bson.D{{"$match", bson.D{
+			{"workflow_runs.phase", model.WorkflowRunStatusSucceeded},
+		}},
+		},
+		bson.D{{"$count", "count"}},
+	}
+
+	// Count Succeeded workflowRuns
```
Comment (Saranya-jena) [inline litmus-portal/graphql-server/pkg/analytics/handler/handler.go:1049]: Can be changed to "// Count Failed workflowRuns"

File: litmus-portal/graphql-server/pkg/analytics/handler/handler.go [hunk 2]
```diff
@@ -966,3 +966,204 @@ func GetScheduledWorkflowStats(projectID string, filter model.TimeFrequency, sho
...
+	failedWorkflowRuns := bson.A{
+		bson.D{{"$match", bson.D{
+			{"workflow_runs.phase", model.WorkflowRunStatusFailed},
+		}},
+		},
+		bson.D{{"$count", "count"}},
+	}
+
+	// Count Succeeded workflowRuns
```
Comment (Saranya-jena) [inline litmus-portal/graphql-server/pkg/analytics/handler/handler.go:1058]: Can be changed to "// Count Running workflowRuns"

File: litmus-portal/graphql-server/pkg/analytics/handler/handler.go (hunk 1 above)
Comment (SarthakJain26) [reply litmus-portal/graphql-server/pkg/analytics/handler/handler.go]: Done

File: litmus-portal/graphql-server/pkg/analytics/handler/handler.go (hunk 2 above)
Comment (SarthakJain26) [reply litmus-portal/graphql-server/pkg/analytics/handler/handler.go]: Done

**Review by vanshBhatia-A4k9 — APPROVED** [review]
Thank you for the efforts and the new API :rocket:

**Review by Saranya-jena — APPROVED** [review]
LGTM. thanks 🚀

**Pull request merged**